### PR TITLE
new package: zathura-pdf-poppler at 0.2.9

### PR DESCRIPTION
**Pull request by tristelune1**
## Description of package

Plugin to view pdf with zathura. 

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/zathura-pdf-poppler/build [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+meson build \
+	--prefix=/usr \
+	--sysconfdir=/etc \
+	--buildtype=release \
+	--backend=ninja
```
Comment (dylanaraps) [inline community/zathura-pdf-poppler/build]: Is `--backend=ninja` needed?

File: community/zathura-pdf-poppler/depends [hunk 2]
```diff
@@ -0,0 +1,2 @@
+poppler-glib
+zathura
```
Comment (dylanaraps) [inline community/zathura-pdf-poppler/depends]: Missing `meson make` and `ninja make`.

**Comment by tristelune1** [comment]
Done. I did the following changes:

- dependencies modified. As I saw ninja is the backend. 
- indentation set to 4 spaces.

File: community/zathura-pdf-poppler/depends [hunk 3]
```diff
@@ -0,0 +1,4 @@
+meson make
+ninja make
```
Comment (dylanaraps) [inline community/zathura-pdf-poppler/depends]: This line can now be removed due to my changes here: https://github.com/kisslinux/repo/commit/e27b638dda62dd9f090c978507d01fa5237e5a45

**Comment by tristelune1** [comment]
I've done it and removed the ninja backend in the `build` file.

**Pull request merged**